gradle.properties

```diff
@@ -1,2 +1,2 @@
 junitVersion=4.12
-artifactVersion=1.0.0
+artifactVersion=1.0.1-beta1
```

Comment:
One option for this could be to update build.gradle's version line with something like:

    version "${System.env.TRAVIS_TAG ?: $artefactVersion}"